README.md

```diff
@@ -218,6 +218,14 @@ the mnemonic and the cadr is any valid mugur-key.
 ```
 
 ## Insert raw C code (advanced users)
+In case you have custom keycodes that you have created or simply want to
+insert a keycode that may be covered by mugur you can use the RAW mugur-key.
+
+| mugur-key        | example comment                       |
+|:-----------------|:--------------------------------------|
+| (RAW "KC_MYKEY") | Insert your unmodified custom keycode |
```

Comment:
I'm still failing to see what a custom keycode would look like. Please give an example that you've personally used.

Reply:
Personally, I've used RAW to refer a few missing keycodes, (DT_PRNT, DT_UP/DOWN, KC_ASRP/ASUP/ASDN) as well as functions in Tap Dances. However, I'm still working on trying to incorporate `ACTION_TAP_DANCE_LAYER_MOVE` and `ACTION_TAP_DANCE_LAYER_TOGGLE` so I haven't finished updating that PR quite yet.

But you can see more about custom keycodes here: https://github.com/qmk/qmk_firmware/blob/master/docs/custom_quantum_functions.md